README.md

```diff
@@ -27,6 +27,7 @@ And the `Highlighter` will mark all occurrences of search terms within the text:
 | highlightClassName | String |  | CSS class name applied to highlighted text |
 | highlightStyle | Object |  | Inline styles applied to highlighted text |
 | sanitize | Function |  | Process each search word and text to highlight before comparing (eg remove accents); signature `(text: string): string` |
+| escape | Boolean |  | Escape characters which are meaningful in regular expression |
```

Comment:
nit: I think `autoEscape` might be a better prop-name for this. "escape" sounds more like an _action_ (aka a method name) to me.